issues with json-loader

Hi, first of all compliment for the component, really nice. Running it with webpack and in specific using electrode.io I'm getting this error:

```

./~/react-phone-number-input/index.common.js
Module not found: Error: Cannot resolve module 'libphonenumber-js/metadata.min'
 @ ./~/react-phone-number-input/index.common.js 4:15-56
./~/react-phone-number-input/build/input.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./country names in 
 @ ./~/react-phone-number-input/build/input.js 65:20-46

```

I have been able to fix it adding the ext to the files when imported. example:

```
import country_names from './country names'
```
with
```
import country_names from './country names.json'
```

how explained here https://github.com/webpack-contrib/json-loader

If you think that is a good fix here the PR #12 

thank you

--- a/index.common.js
+++ b/index.common.js
@@ -1,7 +1,7 @@
 'use strict'
 
 var React = require('react')
-var metadata = require('libphonenumber-js/metadata.min')
+var metadata = require('libphonenumber-js/metadata.min.json')
 
 var Input = require('./custom').default
 

--- a/index.es6.js
+++ b/index.es6.js
@@ -1,5 +1,5 @@
 import { createElement } from 'react'
-import metadata from 'libphonenumber-js/metadata.min'
+import metadata from 'libphonenumber-js/metadata.min.json'
 
 import CustomInput from './es6/input'
 

--- a/runnable/generate-countries.js
+++ b/runnable/generate-countries.js
@@ -1,4 +1,4 @@
-import country_names from '../source/international/country names'
+import country_names from '../source/international/country names.json'
 
 import path from 'path'
 import fs from 'fs'

--- a/source/input.js
+++ b/source/input.js
@@ -8,7 +8,7 @@ import classNames from 'classnames'
 // just to reduce the overall bundle size.
 import { Select } from './react-responsive-ui'
 
-import country_names from './country names'
+import country_names from './country names.json'
 import International_icon from './international icon'
 
 // A list of all country codes

